File: LeetCode/0709. To Lower Case/solution.cpp
/**
 * 114 / 114 test cases passed.
 * Runtime: 0 ms
 * Memory Usage: 6 MB 
 */
class Solution {
public:
    string toLowerCase(string s) {
        for (auto &c : s) {
            if ('A' <= c && c <= 'Z') {
                c += 32;
            }
        }
        return s;
    }
};

File: LeetCode/0709. To Lower Case/solution.py
"""
114 / 114 test cases passed.
Runtime: 28 ms
Memory Usage: 15.1 MB
"""
class Solution:
    def toLowerCase(self, s: str) -> str:
        return ''.join([c.lower() if 'A' <= c <= 'Z' else c for c in s])
    
